Source/Meadow.Foundation.Peripherals/Sensors.Temperature.Pct2075/Samples/Pct2075_Sample/meadow.config.yaml

```diff
@@ -0,0 +1,2 @@
+﻿MonoControl:
+    Options: --jit
```

Comment:
This is on by default. I vote we remove the config file for samples so its less cluttered.